client/src/helpers/APICalls/login.ts

```diff
@@ -1,6 +1,22 @@
-import { AuthApiData } from '../../interface/AuthApiData';
+import { AuthApiData, DemoAuthData } from '../../interface/AuthApiData';
 import { FetchOptions } from '../../interface/FetchOptions';
 
+export const demoLogin = async (email: string, password: string): Promise<DemoAuthData> => {
+  const fetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ email, password }),
+  };
+
+  return await fetch(`${process.env.PUBLIC_URL}/auth/demo-login`, fetchOptions)
+    .then((res) => {
+      return res.json();
+    })
+    .catch(() => ({
+      error: { message: 'Unable to connect to server in demo mode. Please try again' },
+    }));
```

Comment:
I actually find this fine! AS fetch requires us to do res.json() on incoming response body. 😄 Good work